x-pack/plugins/infra/server/lib/alerting/inventory_metric_threshold/inventory_metric_threshold_executor.ts

```diff
@@ -193,6 +193,9 @@ export const createInventoryMetricThresholdExecutor = (libs: InfraBackendLibs) =
           libs.basePath
         );
         scheduledActionsCount++;
+
+        const additionalContext = results && results.length > 0 ? results[0][group] : null;
```

Comment:
What are `results` and why do we only get the first item?
It would help if you have an example for results.

Reply:
The results array holds result of each condition evaluated. If the rule has 3 conditions, results array will have result of each condition - alert/warning/no data/error etc. Since we are only interested in the common fields in the bucket, we can use any one of the results item.